src/workflows-app/SubmissionDetails.js

```diff
@@ -392,9 +393,6 @@ export const BaseSubmissionDetails = ({ name, namespace, workspace, submissionId
                           },
                         },
                       ],
-                      styleCell: ({ rowIndex }) => {
```

Comment:
Out of curiosity do you know what was the purpose of this styling? Was it for accessiblity?

Reply:
I thought that it may have been, but I wasn't seeing any accessibility errors, nor did I see anything change on the page. I tried to git blame to ask whoever may have implemented it in CBAS UI, but it said it was me! :joy: